core/base/src/TString.cxx

```diff
@@ -2288,7 +2299,7 @@ TObjArray *TString::Tokenize(const TString &delim) const
 void TString::FormImp(const char *fmt, va_list ap)
 {
    Ssiz_t buflen = 20 + 20 * strlen(fmt);    // pick a number, any strictly positive number
-   Clobber(buflen);
+   buflen = Clobber(buflen);
```

Comment:
Maybe a comment here to clarify the intent of changing the value of the buflen variable for the rest of the function (since it wasn't done previously)?